src/rascal/math/bessel.hh

```diff
@@ -199,6 +252,7 @@ namespace rascal {
         if (n_up > 0) {
           this->upward_recursion(distance, fac_a, n_up);
         }
+        assert(this->bessel_values.isFinite().all());
```

Comment:
CI is failing on this assert, so it looks like it was useful!

Reply:
yes I used them a lot too when making the thing work. The problem on the CI is still mysterious though.